### ISSUE TITLE
Tokelau

### ISSUE THREAD

**Issue by tmtmtmtm** [everypolitician/everypolitician-data#688]

**Comment by confirmordeny** [comment]
The Council of Government (Cabinet) members are listed on the right hand side of this page: http://www.tokelau.org.nz/

This looks like the Executive though and not the legislature.

**Comment by tmtmtmtm** [comment]
Yeah — finding the members of the Fono has been rather more difficult… 

I searched a couple of big New Zealand news portals last week to see if I could find stories from around the election, but very few even reported on the election at all, and none that I could find listed any other members.

**Comment by appleorange1** [comment]
I found the General Fono delegates!

See pages 12-13 of this document:
http://www.tokelau.org.nz/site/tokelau/files/2016Docs/GF-Decisions10-11mar16E.pdf

However, to make things more confusing, the Faipules and the Pulenukus only have their last names given. That's okay, though, because their full names can be found on this candidate list:

http://www.tokelau.org.nz/Bulletin/Articles+archive+2013/Tokelau+Candidates+for+2014+Election+confirmed.html
